db/migrate/20150306060237_unread_migration.rb

```diff
@@ -0,0 +1,16 @@
+class UnreadMigration < ActiveRecord::Migration
+  def self.up
+    create_table :read_marks, :force => true do |t|
+      t.integer  :readable_id,   foreign_key: false
+      t.integer  :user_id,       :null => false
+      t.string   :readable_type, :null => false, :limit => 20
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Put one space between the method name and the first argument.